dev.sh

```diff
@@ -1,16 +1,16 @@
 #!/bin/bash
 # ----------------------------------------------------------------
-# Build docker dev stage and add local code for live development 
+# Build docker dev stage and add local code for live development
 # ----------------------------------------------------------------
 
 # Build docker image up to dev stage
 DOCKER_BUILDKIT=1 docker build \
--t imu_humble \
--f Dockerfile --target dev .
+    -t imu_humble \
```

Comment:
Please change this for `av_imu_microstrain_launch_humble`

Reply:
Done